tests/common/reboot.py

```diff
@@ -169,7 +169,7 @@ def execute_reboot_helper():
         raise Exception('DUT {} did not shutdown'.format(hostname))
 
     if not wait_for_ssh:
-        return
+        return dut_datetime
```

Comment:
was this return type change checked for any other test using this reboot function?

Reply:
the other callers(with wait_for_ssh=false) is not getting this return value, so behavior won't change for other callers.

Reply:
might be cleaner to have the same type and value being returned by function on all scenarios, like even when wait_for_ssh = true